packages/server/modules/automate/utils/automateFunctionRunStatus.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  AutomationRunStatus,
+  AutomationRunStatuses
+} from '@/modules/automate/helpers/types'
+import { FunctionRunReportStatusesError } from '@/modules/automate/errors/runs'
+import { AutomateRunStatus } from '@/modules/core/graph/generated/graphql'
+
+const AutomationRunStatusOrder: Array<AutomationRunStatus | AutomationRunStatus[]> = [
```

Comment:
maybe a simple object, with keys RunStatus and numbers as priority would make the implementation simpler

Reply:
I dig it, done

Reply:
nice